tests/common/helpers/bgp.py

```diff
@@ -164,7 +199,12 @@ def announce_route(self, route):
         url = "http://%s:%d" % (self.ptfip, self.port)
         resp = requests.post(url, data={"commands": msg}, proxies={"http": None, "https": None})
         logging.debug("announce return: %s", resp)
-        assert resp.status_code == 200
+        assert resp.status_code == 200, (
+            "Expected HTTP 200 from exabgp API, but got {}. Response: {}."
+        ).format(
+            resp.status_code,
+            resp.text
```

Comment:
Not sure resp has text, so, it's better to remove it and just print status_code out.